.github/workflows/ci.yml

```diff
@@ -121,9 +132,7 @@ jobs:
       - run: |
           julia --project=docs -e '
             using Pkg
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRIFiles"))
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRICore"))
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRIPlots"))
+            Pkg.develop([PackageSpec(path=pwd(), subdir="KomaMRIBase"), PackageSpec(path=pwd(), subdir="KomaMRICore"), PackageSpec(path=pwd(), subdir="KomaMRIFiles"), PackageSpec(path=pwd(), subdir="KomaMRIPlots")])
```

Comment:
There must be something weird here, as the docs are not being generated correctly.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/13047bca75e623326300110a2ecd54b35fcffb21